jest.a11y.config.js

```diff
@@ -1,12 +1,15 @@
 module.exports = {
   roots: ['<rootDir>/src/test/a11y'],
-  testRegex: "(/src/test/.*|\\.(test|spec))\\.(ts|js)$",
-  testEnvironment: "node",
+  "testRegex": "(/src/test/.*|\\.(test|spec))\\.(ts|js)$",
+   "moduleFileExtensions": [
+    "ts",
+    "js"
+  ],
+  "testEnvironment": "node",
   transform: {
     '^.+\\.ts?$': 'ts-jest',
   },
-  moduleFileExtensions: ['ts', 'js'],
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
```

Comment:
If you havent changed this file please revert to how it was, you need to be aware of what youre commiting and what youre not commiting

Reply:
from the comments you made you mentioned you havent purposefully changed code, so just changing the previous comment was only part of the problem, you have to think pragmatically and forward think

Reply:
As you mentioned, I reverted all the changes to that file. Can be checked again. thanks,